alpha/apps/kaltura/lib/kESearchUtils.php

```diff
@@ -0,0 +1,39 @@
+<?php
+class kESearchUtils
+{
+	const ELASTIC_CLUSTER_NAME = 'elastic_cluster_name';
+	const ELASTIC_CLUSTER_NAME_TTL = 60;
+	
+	public static function getElasticClusterName()
+	{
+		$cache = kCacheManager::getSingleLayerCache(kCacheManager::CACHE_TYPE_ELASTIC_EXECUTED_CLUSTER);
+		if ($cache)
+		{
+			$elasticClusterName = $cache->get(self::ELASTIC_CLUSTER_NAME);
+			
+			if ($elasticClusterName)
+			{
+				KalturaLog::log("Returning value from memcache [$elasticClusterName]");
+				return $elasticClusterName;
+			}
+		}
+		
+		$elasticClient = new elasticClient();
+		$elasticClusterName = $elasticClient->getElasticClusterName();
+		unset($elasticClient);
+		
+		$clusterToServer = kConf::get('cluster_name_to_server', 'elastic', array());
```

Comment:
in nvp1 (also in my aio machine) the elastic cluster name and the sphinx_log_server.server name are different
So that's the mapping in elastic.ini:
[cluster_name_to_server]
esearch7-a = esearch-7a 
esearch7-b = esearch-7b

Reply:
We decided we will update the sphinx_log_server.server names to match elastic cluster_name
So no need to translate between them.